notebooks/dc2/truth_variable_info.ipynb

```diff
@@ -0,0 +1,520 @@
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d_summary[[f\"flux_{band}\" for band in \"ugrizy\"]].min()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Like earlier, we will now use the summary table to select variable stars near a certain coordinate. Here, we further select bright stars and stars with enough variability (to ensure that they appear in the variability table). We will store the selected stars in `objects_to_plot` which we will use later to extract and plot the their light curves."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "selected = SkyCoord(d_summary[\"ra\"].values, d_summary[\"dec\"].values, unit=\"deg\").separation(SkyCoord(60.5, -36.6, unit=\"deg\")).arcmin < 2\n",
+    "selected &= d_summary[\"flux_r\"] > 1.0e3\n",
+    "selected &= d_summary[\"above_threshold\"] > 0\n",
+    "\n",
+    "objects_to_plot = d_summary.iloc[np.flatnonzero(selected)]\n",
+    "objects_to_plot"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "### Loading supernova light curve (variability) table\n",
+    "\n",
+    "We will now load the \"variability table\" for stars."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "star_lc_cat = GCRCatalogs.load_catalog(\"desc_dc2_run2.2i_truth_star_variability\")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Again, recall that in the \"variability table\", each row corresponds to one measurement, and we need to collect all the rows that have `id` values overlap with `objects_to_plot[\"id\"].values`. \n",
+    "\n",
+    "Also remember that the variability table is a large table (even larger in the star case). We will use the same `filters` trick again to reduce the memory footprint, but the cell below will still result in ~4 GB at peak memory usage. It will take a while to run. If runtime is a concern, you may want to use the SQL database that we also provide to access these information instead."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "d_lc = pd.DataFrame(star_lc_cat.get_quantities(\n",
+    "    ['id', 'MJD', 'bandpass', 'delta_flux'],\n",
+    "    filters=[GCRQuery((lambda x: np.isin(x, objects_to_plot[\"id\"].values), \"id\"))],\n",
+    "))\n",
+    "d_lc.head()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Once we collect the light curve infomation in `d_lc`. We can use the same `groupby` trick to split the data frame by ID and bandpass, and proceed to make light curve plot."
```

Comment:
Add "a" or "the" before "light curve plot".    E.g.,
..and proceed to make the light curve plot